contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--3.1.0--3.2.0.sql

```diff
@@ -6,5 +6,17 @@ SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false)
 DROP CAST IF EXISTS(NUMERIC AS sys.BIT);
 CREATE CAST (NUMERIC AS sys.BIT) WITH FUNCTION sys.numeric_bit (NUMERIC) AS IMPLICIT;
 
+DROP FUNCTION IF EXISTS sys.varchar2money(sys.VARCHAR) CASCADE;
+
```

Comment:
Tried with below code as well. ALTER function not working. Same goes for sys.babelfish_drop_deprecated_object.
Found similar implementation of DROP function in contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--2.4.0--3.0.0.sql (DROP FUNCTION sys.babelfish_update_server_collation_name();)

ALTER FUNCTION sys.varchar2money(sys.VARCHAR) RENAME TO varchar2money_deprecated_3_2;
CALL sys.babelfish_drop_deprecated_object('function', 'sys', 'varchar2money_deprecated_3_2');

Reply:
cast is dependent on function  sys.varchar2money